src/trajectory.jl

```diff
@@ -51,7 +87,7 @@ $(TYPEDEF)
 
 Samples the end-point of the trajectory.
 """
-struct EndPointTS <: AbstractTrajectorySampler end
+struct MetropolisTS <: AbstractTrajectorySampler end
```

Comment:
This makes sense. We should warn the user though since this is breaking.

Reply:
It was actually suggested by you :smile: I can make sure to add a deprecation for this.